### DISCUSSION TITLE
%E6%95%B0%E5%AD%A6/%E5%A4%8D%E5%8F%98%E5%87%BD%E6%95%B0/%E8%A7%A3%E6%9E%90%E5%87%BD%E6%95%B0%E7%9A%84%E9%AB%98%E9%98%B6%E5%AF%BC%E6%95%B0/

### DISCUSSION THREAD

**Discussion by giscus** [UNLINEARITY/Learn-Everything#75] [Q&A] [answered]
# %E6%95%B0%E5%AD%A6/%E5%A4%8D%E5%8F%98%E5%87%BD%E6%95%B0/%E8%A7%A3%E6%9E%90%E5%87%BD%E6%95%B0%E7%9A%84%E9%AB%98%E9%98%B6%E5%AF%BC%E6%95%B0/

https://www.unlinearity.top/%E6%95%B0%E5%AD%A6/%E5%A4%8D%E5%8F%98%E5%87%BD%E6%95%B0/%E8%A7%A3%E6%9E%90%E5%87%BD%E6%95%B0%E7%9A%84%E9%AB%98%E9%98%B6%E5%AF%BC%E6%95%B0/

**Comment by yule153604** [comment, marked as answer]
上面的没有渲染成功

$$
f(z_{0})= \dfrac{1}{2\pi i} \oint_{C} \dfrac{f(z)}{z-z_{0}}dz
$$

$$
\oint_{C} \dfrac{f(z)}{z-z_{0}}dz=2\pi i \cdot f(z_{0})
$$
